File: PoCCollectEats/src/app/api/models/choixoption.ts
/* tslint:disable */
/* eslint-disable */
import { Optionsplat } from './optionsplat';
import { Plat } from './plat';
export interface Choixoption {
  id?: number;
  nomchoix?: null | string;
  optionsplats?: null | Array<Optionsplat>;
  plats?: null | Array<Plat>;
  surcout?: null | number;
}

File: CollectEats/CollectEats/Models/Choixoption.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace CollectEats.Models;

public partial class Choixoption
{
    public int Id { get; set; }

    public string? Nomchoix { get; set; }

    public decimal? Surcout { get; set; }

    public virtual ICollection<Optionsplat> Optionsplats { get; set; } = new List<Optionsplat>();

    [JsonIgnore]
    public virtual ICollection<Plat> Plats { get; set; } = new List<Plat>();
}
